MVVM/app/src/main/java/com/lee989898/mvvm/LoginSuccessActivity.kt

```diff
@@ -0,0 +1,19 @@
+package com.lee989898.mvvm
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import androidx.databinding.DataBindingUtil
+import com.lee989898.mvvm.databinding.ActivityLoginSuccessBinding
+
+class LoginSuccessActivity : AppCompatActivity() {
+
+    private lateinit var binding: ActivityLoginSuccessBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = DataBindingUtil.setContentView(this, R.layout.activity_login_success)
+
+        val getEmail = intent.getParcelableExtra<EmailData>("email")
+        binding.email.text = getEmail?.email
```

Comment:
```suggestion
        intent.getParcelableExtra<EmailData>(EMAIL)?.let { email ->
                binding.email.text = email
        }
```

이건 그냥 참고사항입니다! (굳이 코드를 변경하지 않아도 된다는 뜻~) 
변수없이 `let`을 사용할 수도 있습니다~!

Reply:
오 훨씬 깔끔하네요!!